bills-app/src/main/webapp/index.html

```diff
@@ -5,14 +5,19 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="api.js"></script> 
   </head>
-  <body>
+  <body onload = "addCode()">
     <div id="content">
       <h1>My Portfolio</h1>
       <p>This is my portfolio.</p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+
+      <p>ProPublica API Status:</p>
+      <div id = "API-status-container"></div>
```

Comment:
nit, to match the rest of the file
```suggestion
      <div id="API-status-container"></div>
```